.eslintrc

```diff
@@ -8,11 +8,15 @@
     "plugin:jest/recommended",
     "plugin:cypress/recommended"
   ],
-  "plugins": ["jsdoc"],
+  "plugins": ["jsdoc", "@typescript-eslint"],
```

Comment:
Looks like this should have been added before because it was causing the eslint action to fail with `Definition for rule '@typescript-eslint/no-unused-vars' was not found`.

Reply:
That's because the typescript eslint plugins/rules are on an override. I've moved the rule to the override: https://github.com/frontity/frontity/pull/506/commits/8bcd609180a09c22959a9c04030fd21f65d4e645

Reply:
oh gotcha 👍